lib/server.js

```diff
@@ -122,7 +122,7 @@ class S3Server {
             monitoringClient.httpRequestsTotal.labels(labels).inc();
             monitoringClient.httpRequestDurationSeconds
                 .labels(labels)
-                .observe(responseTimeInMs / 1000000);
+                .observe(responseTimeInMs / 1000000000);
```

Comment:
Use exponent notation so we don't make that mistake again :slightly_smiling_face: 

```
❯ node
> 1e9
1000000000
```